fdbserver/KeyValueStoreSQLite.actor.cpp

```diff
@@ -760,9 +760,6 @@ struct RawCursor {
 				// Number of bytes wasted in the fragmented case (for the extra key copies)
 				int fragmentedWaste = kv.key.size() * (fragments - 1);
 
-				// Total bytes used for the fragmented case
-				//int fragmentedTotal = kv.expectedSize() + fragmentedWaste;
```

Comment:
fragmentedTotal is referenced at line 772 below in commented code. Maybe keep commented as is?

Reply:
Done